login-config.yaml

```diff
@@ -0,0 +1,50 @@
+# API documentation configuration file
+# Used for https://vipps.no/developer-pages
+# This file will be fetched by the user's browser
+# Do not store personal or confidental information in this file
+​
```

Comment:
There is an invisible character in each of the empty lines. Maybe it is your IDE that put it there, but the `js-yaml` module cannot decode it this way. Could you check please?

Reply:
Found the issue, committing now.